gnocchi/metric/v1/resources/doc.go

```diff
@@ -31,5 +31,51 @@ Example of Getting a resource
 		panic(err)
 	}
 
+Example of Creating a resource without a metric
+
+	createOpts := resources.CreateOpts{
+		ID: "23d5d3f7-9dfa-4f73-b72b-8b0b0063ec55",
+	}
+	resourceType = "generic"
+	resource, err := resources.Create(gnocchiClient, resourceType, createOpts).Extract()
+	if err != nil {
+		panic(err)
+	}
+
+Example of Creating a resource with links to some existing metrics with an ending timestamp of the resource
```

Comment:
I think this might be a bug with Gnocchi itself.

If no start date is specified, Gnocchi uses the current time.

If both a start and end date are given, Gnocchi will ensure the start date is earlier than the end date. 

But if only an end date is specified, Gnocchi will still use the current time, which means the start date will come _after_ the end date.

Again, I think this is a bug in Gnocchi. But because of it, we probably should not use it as an example :)